source/routing/defining-your-routes.md

```diff
@@ -45,335 +45,48 @@ the case of `/`, the name `index`).
 The `{{link-to}}` helper will also add an `active` class to the link that
 points to the currently active route.
 
-You can customize the behavior of a route by creating an `Ember.Route`
-subclass. For example, to customize what happens when your user visits
-`/`, create an `route:index`:
-
-```bash
-ember generate route index
-```
-
-```app/routes/index.js
-Ember.Route.extend({
-  setupController(controller) {
-    // Set the IndexController's `title`
-    controller.set('title', 'My App');
-  }
-});
-```
-
-`controller:index` is the starting context for the `index` template.
-Now that you've set `title`, you can use it in the template:
+## Nested Routes
 
-```handlebars
-<!-- get the title from the IndexController -->
-<h1>{{title}}</h1>
-```
+Often you'll want to have a template that displays inside another template.
+For example, in a blogging application, instead of going from a list of blog
+posts to creating a new post, you might want to have the post creation page
+display next to the list.
 
-(If you don't explicitly define an `controller:index`, Ember.js will
-automatically generate one for you.)
-
-Ember.js automatically figures out the names of the routes and controllers based on
-the name you pass to `this.route`.
-
-<table>
-  <thead>
-  <tr>
-    <th>URL</th>
-    <th>Route Name</th>
-    <th>
-      Controller<br/>
-      <code>app/controllers/</code>
-    </th>
-    <th>
-      Route<br/>
-      <code>app/routes/</code>
-    </th>
-    <th>
-      Template<br/>
-      <code>app/templates/</code>
-    </th>
-  </tr>
-  </thead>
-  <tr>
-    <td><code>/</code></td>
-    <td><code>index</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.js</code></td>
-    <td>↳<code>index.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/about</code></td>
-    <td><code>about</code></td>
-    <td>↳<code>about.js</code></td>
-    <td>↳<code>about.js</code></td>
-    <td>↳<code>about.hbs</code></td>
-  </tr>
-  <tr>
-    <td><code>/favs</code></td>
-    <td><code>favorites</code></td>
-    <td>↳<code>favorites.js</code></td>
-    <td>↳<code>favorites.js</code></td>
-    <td>↳<code>favorites.hbs</code></td>
-  </tr>
-</table>
-
-### Nested Routes
+In these cases, you can use **nested routes** to display one template inside
```

Comment:
`you can nest routes to display`

Unsure nested routes is first-class enough to get bold etc.